### ISSUE TITLE
Regression in viewbox size

### ISSUE THREAD

**Issue by claudep** [deeplook/svglib#190]
Commit 82860fa54f8a2b added a regression in master when a viewbox contains percentage height/width values (most files in the SVG Tiny test suite are affected).

This is due to the fact that svglib now takes viewbox width and height into account (which is a good thing), however master is currently not handling percentage values. To fix that, #186 is adding percentage value support. @deeplook Are you ok to merge that PR?

FYI, the test which currently fails:
```
diff --git a/tests/test_basic.py b/tests/test_basic.py
index 1d293c4..9fed4bb 100755
--- a/tests/test_basic.py
+++ b/tests/test_basic.py
@@ -651,6 +651,19 @@ class TestViewBox(object):
         # Main group coordinates are translated to match the viewBox origin
         assert drawing.contents[0].transform == (1, 0, 0, -1, 60.0, 40.0)
 
+    def test_percent_width_height(self):
+        drawing = svglib.svg2rlg(io.StringIO(textwrap.dedent(u'''\
+            <?xml version="1.0"?>
+            <svg xmlns="http://www.w3.org/2000/svg"
+                 width="100%" height="100%"
+                 viewBox="0 0 480 360">
+                <g fill="#E70013">
+                    <rect x="60" y="40" width="120" height="80"/>
+                </g>
+            </svg>
+        ''')))
+        assert (drawing.width, drawing.height) == (480, 360)
+
 
 class TestEmbedded(object):
     def test_svg_in_svg(self):
```

**Comment by deeplook** [comment]
Sure, go! And thanks!

**Comment by claudep** [comment]
Fixed by 719e9c7e6643189a
